crypto/fipsmodule/rand/new_rand.c

```diff
@@ -156,29 +156,20 @@ static void rand_get_ctr_drbg_seed_entropy(
 }
 
 // rand_ctr_drbg_reseed reseeds the CTR-DRBG state in |state|.
-static void rand_ctr_drbg_reseed(struct rand_thread_local_state *state) {
+static void rand_ctr_drbg_reseed(struct rand_thread_local_state *state,
+  uint8_t seed[CTR_DRBG_ENTROPY_LEN],
+  uint8_t personalization_string[CTR_DRBG_ENTROPY_LEN],
+  size_t personalization_string_len) {
 
   GUARD_PTR_ABORT(state);
```

Comment:
Should `seed` and `personalization_string` be guarded for NULL or do we enforce that earlier on higher in the function call stack? Since these are just passed as pointers technically, not sure if the compiler would catch these otherwise?

Reply:
I only use array types when calling these functions. so in practice no problem.